plugins/linear-genome-view/src/LinearGenomeView/model.ts

```diff
@@ -243,6 +243,15 @@ export function stateModelFactory(pluginManager: PluginManager) {
          * show the "gridlines" in the track area
          */
         showGridlines: true,
+
+        /**
+         * #property
+         * highlights on the LGV from the URL parameters
+         */
+        highlight: types.optional(
+          types.frozen<ParsedLocString>(),
+          {} as ParsedLocString,
```

Comment:
I believe this cast may not be valid, as refName is required on ParsedLocString. potentially, making it something like {assemblyName:string,refName:string,start:number,end:number}|undefined might be better than parsedlocstring, because then you know you have all required fields, or else it is just undefined